insights/parsers/lspci.py

```diff
@@ -85,7 +85,17 @@ def get_pci_dev(self, dev_name):
             (dict): Returns device details along with 'Subsystem', 'Kernel Driver in Use', 'Kernel Modules'.
             Returns `None` if device doesn't exists
         """
-        return self.pci_dev_details.get(dev_name, None)
+        return self.data.get(dev_name, None)
+
+    @property
+    def get_pci_devs(self):
```

Comment:
The keyword `get_`, as a `verb` is often used by function name, for an attribute/property, better use a noun, how do you think?

Reply:
pci_dev_list will be better for thsi